nyt-top-20-science-news/src/components/App/App.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import Container from 'react-bootstrap/Container';
+import Footer from '../Footer';
+import Header from '../Header';
+import NewsFeed from '../NewsFeed';
+import SearchBox from '../SearchBox';
+
+/**
+ * Main application
+ */
+function App() {
+  const [news, setNews] = useState([]);
+  const [filteredNews, setFilteredNews] = useState(null);
+
+  const TOP_NEWS_ENDPOINT = 'https://api.nytimes.com/svc/topstories/v2/science.json?api-key=';
+  const ARTICLE_COUNT_LIMIT = 20;
+
+  /**
+   * Retrieve NYT Top Science Stories
+   */
+  async function retrievePopularStories() {
+    try {
+      const response = await fetch(TOP_NEWS_ENDPOINT + process.env.REACT_APP_NYT_API_KEY);
```

Comment:
elegant approach to obfuscating secure data in a way that would be easy to deploy on a serverless/static platform with "secrets"